### ISSUE TITLE
Notebook: Beyond Classical Search - 3e

### ISSUE THREAD

**Issue by jsuyash1514** [aimacode/aima-java#444]
@samagra14 Currently, there is no notebook illustrating the use of local search and online search. I would like to work on the same. Should I proceed?

**Comment by samagra14** [comment]
Yeah sure. Go ahead. Keep mentioning the sections you are adding.

**Comment by ghost** [comment]
@samagra14 
Can I work on this notebook to add 
HILL-CLIMBING
SIMULATED-ANNEALING
GENETIC-ALGORITHM
AND-OR-GRAPH-SEARCH
Can I proceed?

**Comment by jsuyash1514** [comment]
@vksngh19 I've already added these algorithms in the notebook. I'll be adding the rest algorithms of this chapter and will open a PR soon.
